app/helpers/abr_state_methods/az.rb

```diff
@@ -111,11 +116,37 @@ def election_selection=(val)
       self.send("#{self.class.make_method_name("Every Election I authorize the County Recorder to include my name on the PEVL and automatically send")}=", "On")
     end
   end
+
+  def identification_data
+    case self.identification_selection
+    when "drivers_license_state"
+     return (self.drivers_license_state())
+    when "last_4_ssn"
+      return (self.last_4_ssn())
+    when "place_of_birth"
+      return (self.place_of_birth())
+    else
+      return "Missing identification"
+    end
+  end
+
+
+
+
+
+
+
+    
   
   def custom_form_field_validations
     if self.has_mailing_address.to_s == "1"
       custom_validates_presence_of("Mailing_Address")
     end
+
+    if self.identification_selection
```

Comment:
Only thing I'd change is make this `if !self. identification_selection.blank?` because "" evaluates to true in ruby.